src/components/TableBodyRow.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+const TableBodyRow = ({ content }) => {
```

Comment:
Good idea making this it's own component! Really good quality code in here too well done